pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -728,9 +734,19 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 
 	ip := iputil.LastUsableIP(*svcNet)
 	ipStr := ip.String()
-	log.Printf("Ensuring services subnet with %s CIDR and %s gateway", conf.ServiceNetwork[0], ipStr)
+	splitSvcNet, vrrpNet := iputil.SplitNet(*svcNet)
+	svcPrefixLen, _ := splitSvcNet.Mask.Size()
+	splitSvcCIDR := fmt.Sprintf("%s/%d", splitSvcNet.IP.String(), svcPrefixLen)
+	vrrpNetStart := iputil.FirstUsableIP(vrrpNet).String()
+	vrrpNetEnd := iputil.LastUsableIP(vrrpNet).String()
+
+	log.Printf("Overriding ServiceNetwork[0] to %s", splitSvcCIDR)
+	conf.ServiceNetwork[0] = splitSvcCIDR
```

Comment:
I know, ideally we would only change that when saving the Status. Let's just treat this as PoC, we still need to see if that works fine.